network/p2p/test/fixtures.go

```diff
@@ -0,0 +1,305 @@
+package p2ptest
```

Comment:
No new logic in this file, mainly migrated some fixtures from `internal` packages.